### ISSUE TITLE
[Premium] Attribute bonuses seem to get stuck on death

### ISSUE THREAD

**Issue by RacerN2O** [Eniripsa96/SkillAPI#362]
My brother's been using an ability that passivily grants attribute bonuses and they seem to stick permanently on death and then stack with themselves.

**Comment by Zicerite** [comment]
Important to know if they stay permanent on relog and server restart. Do they also stick when both of those things happen?

**Comment by Zicerite** [comment]
Seems to fix itself on logoff and logging back on though, for me anyway.

**Comment by RacerN2O** [comment]
Yes, they don't persist through relogs. Restarts most likely gets rid of them as well but I haven't tested that.
